src/core/issues-notes.js

```diff
@@ -118,7 +118,37 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          var labelListSpan = document.createElement('span');
+          Array.from(ghIssue.labels)
+            .forEach((label, index) => {
+              var labelSpan = document.createElement('span');
+              labelSpan.classList.add("respec-gh-label");
+              labelSpan.style.backgroundColor = `#${label.color}`;
+              labelSpan.style.textTransform = 'none';
+              labelSpan.id = `${dataNum}-label-${index}`;
```

Comment:
An issue can appear in the document more than once, so this would lead to duplicate IDs. That would make the document invalid as per the W3C Validator. 

We probably don't want to give these things IDs anyway (I can't think of any use case for having an id).

Reply:
I was actually using the ids in testing :stuck_out_tongue: